node/src/components/block_accumulator/block_acceptor.rs

```diff
@@ -255,14 +255,19 @@ impl BlockAcceptor {
     fn remove_bogus_validators(
         &mut self,
         era_validator_weights: &EraValidatorWeights,
-    ) -> BTreeSet<NodeId> {
+    ) -> Vec<(NodeId, AcceptorError)> {
```

Comment:
Since this is always the same `AcceptorError` variant, I'd rather just put in the bogusness type:

```suggestion
    ) -> Vec<(NodeId, Bogusness)> {
```

(Would also allocate a bit less memory.)